CRM/Financial/BAO/Payment.php

```diff
@@ -319,7 +320,7 @@ public static function filterUntestedTemplateVariables($params) {
    *
    * @return CRM_Financial_DAO_FinancialTrxn
    */
-  public static function recordRefundPayment($contributionId, $trxnData, $updateStatus) {
+  protected static function recordRefundPayment($contributionId, $trxnData, $updateStatus) {
```

Comment:
Umm you missed a spot 
```
CRM//Contribute/BAO/Contribution.php:4017:      $financialTrxn = CRM_Financial_BAO_Payment::recordRefundPayment($contributionId, $trxnsData, $updateStatus);
``` 
which expect it to be public.

Reply:
@monishdeb I don't see that line.. at that line I see
  ```    if (!empty($financialTrxn)) {```

Are you on latest?

Reply:
@eileenmcnaughton yes I am checking on the latest instance heres the line https://github.com/civicrm/civicrm-core/blob/master/CRM/Contribute/BAO/Contribution.php#L4017

Reply:
@monishdeb but isn't that line removed in this pr?

Reply:
Ahh right ... (arghhh)..  sorry my bad :(